app/views/layouts/darkswarm.html.haml

```diff
@@ -13,7 +13,7 @@
     %link{href: "https://fonts.googleapis.com/css?family=Roboto:400,300italic,400italic,300,700,700italic|Oswald:300,400,700", rel: "stylesheet", type: "text/css"}
 
     = yield :scripts
-    %script{src: "//maps.googleapis.com/maps/api/js?libraries=places,geometry"}
+    %script{src: "//maps.googleapis.com/maps/api/js?libraries=places,geometry&key=#{ENV['GOOGLE_MAPS_API_KEY']}"}
```

Comment:
Maybe this should check if a key is set to ensure backwards compatibility. I'm not sure how Google reacts if an old domain declares an empty key.